android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/QuestionnaireConfig.kt

```diff
@@ -35,7 +35,7 @@ data class QuestionnaireConfig(
   val resourceIdentifier: String? = null,
   val resourceType: String? = null,
   val confirmationDialog: ConfirmationDialog? = null,
-  val groupResource: GroupResourceConfig? = null,
+  var groupResource: GroupResourceConfig? = null,
```

Comment:
Why are we converting this to mutable property?

Reply:
The config data models are majorly used to read and hold configuration data and thus should not mutate